### PR TITLE
Update free-programming-books.md

### PR DESCRIPTION
**Pull request by jackdolgin**

### CODE REVIEW

File: free-programming-books.md [hunk 1]
```diff
@@ -519,16 +533,27 @@
...
+* [Planes and Spherical Trigonometry](http://djm.cc/library/Plane_Spherical_Trigonometry_Wentworth_Smith_edited_2.pdf) - George Wentworth and David Eugene Smith (PDF)
 * [Power Programming with Mathematica](http://mathematica.stackexchange.com/questions/16485/are-you-interested-in-purchasing-david-wagners-power-programming-with-mathemat/22724) - David B. Wagner
 * [Probability and Statistics Cookbook](http://statistics.zone)
 * [Probability and Statistics EBook](http://wiki.stat.ucla.edu/socr/index.php/Probability_and_statistics_EBook)
+* [Recreations in Math](http://djm.cc/library/Recreations_in_Mathematics_Licks_edited.pdf) - H. E. Licks (PDF)
+* [Second Course in Algebra](http://djm.cc/library/Second_Algebra_Hawkes_Luby_Touton_edited.pdf) - Herbert E. Hawkes, William A. Luby, Frank C. Touton (PDF)
 * [Seven Sketches in Compositionality: An Invitation to Applied Category Theory](https://arxiv.org/pdf/1803.05316.pdf) - Brendan Fong and David I. Spivak (PDF)
+* [Statistical Thinking for the 21st Century](https://statsthinking21.org/) - Russell A. Poldrack
```
Comment (eshellman) [inline free-programming-books.md:556]: Another book that should be considered for the R section. And linter doesn't like the trailing slash

File: free-programming-books.md [hunk 2]
```diff
@@ -634,6 +659,7 @@
 * [HTTP Succinctly, Syncfusion](https://www.syncfusion.com/resources/techportal/ebooks/http) (PDF, Kindle) (email address *requested*, not required)
 * [HTTP2 Explained](http://daniel.haxx.se/http2/) - Daniel Stenberg
 * [IPv6 for IPv4 Experts](https://sites.google.com/site/yartikhiy/home/ipv6book) - Yar Tikhiy (PDF)
+* [Network Science](http://networksciencebook.com/) - Albert-Laszló Barabási
```
Comment (eshellman) [inline free-programming-books.md:662]: trailing slash. interesting book

File: free-programming-books.md [hunk 3]
```diff
@@ -1422,6 +1448,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Understanding Flexbox: Everything you need to know](https://ohansemmanuel.github.io/uf_download.html) - Ohans Emmanuel
 * [W3.CSS Succinctly](https://www.syncfusion.com/ebooks/w3_css_succinctly) - Joseph D. Booth
 * [Web Audio API](http://chimera.labs.oreilly.com/books/1234000001552) - Boris Smus
+* [Interneting is Hard (But it Doesn't Have to Be)](https://www.internetingishard.com/) - Oliver James
```
Comment (eshellman) [inline free-programming-books.md:1451]: this looks great. trailing slash

File: free-programming-books.md [hunk 4]
```diff
@@ -1562,6 +1589,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Basic JavaScript for the impatient programmer](http://www.2ality.com/2013/06/basic-javascript.html) - Dr. Axel Rauschmayer
 * [Book of Modern Frontend Tooling](https://github.com/tooling/book-of-modern-frontend-tooling)
 * [Crockford's JavaScript](http://www.crockford.com/javascript/) - Douglas Crockford
+* [Deep JavaScript: Theory and techniques](https://exploringjs.com/deep-js/) - Dr. Axel Rauschmayer
```
Comment (eshellman) [inline free-programming-books.md:1592]: ++

File: free-programming-books.md [hunk 5]
```diff
@@ -1574,6 +1602,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [JavaScript Enlightenment](http://www.javascriptenlightenment.com) - Cody Lindley
 * [JavaScript ES6 and beyond](https://github.com/AlbertoMontalesi/JavaScript-es6-and-beyond-ebook) - Alberto Montalesi (PDF, epub)
 * [JavaScript For Cats](http://jsforcats.com)
+* [JavaScript for Impatient Programmers (ES2020 edition)](https://exploringjs.com/impatient-js/) - Dr. Axel Rauschmayer
```
Comment (eshellman) [inline free-programming-books.md:1605]: yep

File: free-programming-books.md (hunk 2 above)
Comment (eshellman) [reply free-programming-books.md:662]: also, translated versions could be added to other lists - maybe create an issue for native speakers to add.

File: free-programming-books.md [hunk 6]
```diff
@@ -519,16 +533,27 @@
 * [Linear Algebra](http://home.iitk.ac.in/~arlal/book/la.pdf) - Dr. Arbind K Lal, Sukant Pati (PDF) (:construction: *in process*)
 * [Linear Algebra](https://www.math.ucdavis.edu/~linear/linear-guest.pdf) (PDF)
 * [Linear Algebra by Jim Hefferon](http://joshua.smcvt.edu/linearalgebra) - Jim Hefferon
+* [Linear Algebra, Infinite Dimensions, and Maple](https://people.math.gatech.edu/~herod/Hspace/Hspace.html) - James Herod
+* [Linear Methods of Applied Mathematics](http://www.mathphysics.com/pde/) - Evans M. Harrell II and James V. Herod
+[Magic Squares and Cubes](http://djm.cc/library/Magic_Squares_Cubes_Andrews_edited.pdf) - W. S. Anderson (PDF)
 * [Mathematical Logic - an Introduction](http://www.ii.uib.no/~michal/und/i227/book/book.pdf) (PDF)
 * [Mathematics, MTH101A](http://home.iitk.ac.in/~psraj/mth101/) - P. Shunmugaraj, IIT Kanpur
+* [Modern Statistics for Modern Biology](https://www.huber.embl.de/msmb/) - Susan Holmes and Wolfgang Huber
```
[the remark this answers is not in the document]
Comment (jackdolgin) [reply free-programming-books.md:541]: It's a good question! It looks like it's more about stats than it is about R

**Comment by eshellman** [comment]
remember to check the Travis CI output - the linter has 4 complaints left to fix.

**Comment by eshellman** [comment]
remember to check the Travis CI output - the linter has 4 complaints left to fix.

File: free-programming-books.md (hunk 2 above)
Comment (jackdolgin) [reply free-programming-books.md]: I actually went ahead and added them!

**Comment by eshellman** [comment]
Welcome to free-programming-books!

**Pull request merged**